system-contract-dapp-playground/src/api/localStorage/index.ts

```diff
@@ -64,3 +64,22 @@ export const getArrayTypedValuesFromLocalStorage = (key: string) => {
     return { err };
   }
 };
+
+/**
+ * @dev clear all HEDERA transaction results cached in localStorage
+ */
+export const clearTransactionCache = () => {
+  // loop through localStorage items
+  if (typeof localStorage !== 'undefined') {
```

Comment:
nit: see if this works 
```suggestion
  if (localStorage) {
```